frontend/src/metabase/reference/databases/DatabaseDetailContainer.jsx

```diff
@@ -0,0 +1,76 @@
+/* eslint "react/prop-types": "warn" */
+import React, { Component } from 'react';
+import PropTypes from "prop-types";
+import { connect } from 'react-redux';
+
+import DatabaseSidebar from './DatabaseSidebar.jsx';
+import SidebarLayout from 'metabase/components/SidebarLayout.jsx';
+import DatabaseDetail from "metabase/reference/databases/DatabaseDetail.jsx"
+
+import * as metadataActions from 'metabase/redux/metadata';
+import * as actions from 'metabase/reference/reference';
+
+import {
+    getDatabase,
+    getDatabaseId,
+    getIsEditing
+} from '../selectors';
+
+
+const mapStateToProps = (state, props) => ({
+    database: getDatabase(state, props),
+    databaseId: getDatabaseId(state, props),
+    isEditing: getIsEditing(state, props)
+});
+
+const mapDispatchToProps = {
+    ...metadataActions,
+    ...actions
+};
+
+@connect(mapStateToProps, mapDispatchToProps)
+export default class DatabaseDetailContainer extends Component {
+    static propTypes = {
+        params: PropTypes.object.isRequired,
+        location: PropTypes.object.isRequired,
+        database: PropTypes.object.isRequired,
+        isEditing: PropTypes.bool
+    };
+
+    async fetchContainerData(){
+        await actions.rFetchDatabaseMetadata(this.props, this.props.databaseId);
```

Comment:
Why this is `rFetch`, not `fetch`? If `r` means something, then why not writing the whole word there